casr/tests/casr_tests/js/crash

```diff
@@ -0,0 +1 @@
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

Comment:
maybe just generate this file and do not store it?

Reply:
Done